src/test_report.py

```diff
@@ -40,7 +40,7 @@ def __init__(self, configs, args):
         self._email_user = os.getenv('EMAIL_USER')
         self._email_pass = os.getenv('EMAIL_PASSWORD')
 
-    def create_email(self, email_content, send_to, send_from, email_subject):
+    def _create_email(self, send_to, send_from, email_content, email_subject):
```

Comment:
It seems a bit more intuitive to me to have `email_subject` before `email_content`. It's nothing big and I may be biased. So it's rather "for consideration" than something crucial. I wouldn't probably even put a remark here if you haven't been changing fucntion's signature.

Reply:
Yes, my main reason for this change was to get the send to/from together and the email subject/content together as well.  But that's a fair point.